### PR TITLE
Update Simplified Chinese Strings

### PR DESCRIPTION
**Pull request by sr093906**
I only handled strings with '' at end. 

The Simplified Chinese strings were last updated 21 days ago, while the source strings were last updated 9 days ago. And comparing them to find out all differences is a thorny task for me. 

I am sorry.

### CODE REVIEW

**Comment by forrestguice** [comment]
Thank you, I appreciate the work.
I plan to merge this in a couple days, and then tag `v0.14.11` after a week or two.
@jamesliu96 please feel free to review changes if you like.

> The Simplified Chinese strings were last updated 21 days ago, while the source strings were last updated 9 days ago. And comparing them to find out all differences is a thorny task for me.
> 
> I am sorry.

Please don't be sorry, I don't actually expect translators to monitor both files. That is asking too much.

I try to keep the files synchronized (same values, relative order, etc). I'll add a `TODO` next to lines that need some kind of attention. Sometimes I'm a little slow to copy values (also marked `TODO`), or will miss something entirely, so let me know if something seems missing or can't be translated and I'll fix it.

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -399,9 +399,9 @@
     <string name="widgetMode1x1_distance">当前距离</string>     <!-- setting -->
     <string name="widgetMode1x1_apsis">下一个远地点或近地点</string>     <!-- setting -->
 
-    <string name="widgetMode3x1_lightmap_large">Lightmap</string>     <!-- setting -->    <!-- TODO -->
-    <string name="widgetMode3x1_lightmap_medium">Lightmap (medium)</string>     <!-- setting -->    <!-- TODO -->
```
Comment (jamesliu96) [inline app/src/main/res/values-zh-rCN/strings.xml:403]: The `)` `U+0029` right parenthesis should be in fullwidth `）` `U+FF09` to keep the punctuation consistent throughout the project

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -399,9 +399,9 @@
...
-    <string name="widgetMode3x1_lightmap_small">Lightmap (small)</string>     <!-- setting -->    <!-- TODO -->
+    <string name="widgetMode3x1_lightmap_large">光照图</string>     <!-- setting -->    
+    <string name="widgetMode3x1_lightmap_medium">光照图 (中等)</string>     <!-- setting -->
```
Comment (jamesliu96) [inline app/src/main/res/values-zh-rCN/strings.xml:403]: The `)` `U+0029` right parenthesis should be in fullwidth `）` `U+FF09` to keep the punctuation consistent throughout the project
```suggestion
    <string name="widgetMode3x1_lightmap_medium">光照图（中等）</string>     <!-- setting -->    
```

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 3]
```diff
@@ -399,9 +399,9 @@
     <string name="widgetMode1x1_distance">当前距离</string>     <!-- setting -->
     <string name="widgetMode1x1_apsis">下一个远地点或近地点</string>     <!-- setting -->
 
-    <string name="widgetMode3x1_lightmap_large">Lightmap</string>     <!-- setting -->    <!-- TODO -->
-    <string name="widgetMode3x1_lightmap_medium">Lightmap (medium)</string>     <!-- setting -->    <!-- TODO -->
-    <string name="widgetMode3x1_lightmap_small">Lightmap (small)</string>     <!-- setting -->    <!-- TODO -->
+    <string name="widgetMode3x1_lightmap_large">光照图</string>     <!-- setting -->    
+    <string name="widgetMode3x1_lightmap_medium">光照图 (中等)</string>     <!-- setting -->    
+    <string name="widgetMode3x1_lightmap_small">光照图 (小)</string>     <!-- setting -->
```
Comment (jamesliu96) [inline app/src/main/res/values-zh-rCN/strings.xml:404]: same as the line above
```suggestion
    <string name="widgetMode3x1_lightmap_small">光照图（小）</string>     <!-- setting -->    
```

**Review by jamesliu96 — COMMENTED** [review]
LGTM despite the better punctuation to keep everything consistent

**Review by jamesliu96 — APPROVED** [review]

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 4]
```diff
@@ -399,9 +399,9 @@
     <string name="widgetMode1x1_distance">当前距离</string>     <!-- setting -->
     <string name="widgetMode1x1_apsis">下一个远地点或近地点</string>     <!-- setting -->
 
-    <string name="widgetMode3x1_lightmap_large">Lightmap</string>     <!-- setting -->    <!-- TODO -->
-    <string name="widgetMode3x1_lightmap_medium">Lightmap (medium)</string>     <!-- setting -->    <!-- TODO -->
-    <string name="widgetMode3x1_lightmap_small">Lightmap (small)</string>     <!-- setting -->    <!-- TODO -->
+    <string name="widgetMode3x1_lightmap_large">光照图</string>     <!-- setting -->    
+    <string name="widgetMode3x1_lightmap_medium">光照图 （中等）</string>     <!-- setting -->
```
Comment (jamesliu96) [inline app/src/main/res/values-zh-rCN/strings.xml:403]: Oh and maybe remove the space between `光照图` and the parentheses please? This is as well to match the punctuational conventions throughout the translation. You may incorporate the suggestion below or just copy & paste it.
```suggestion
    <string name="widgetMode3x1_lightmap_medium">光照图（中等）</string>     <!-- setting -->    
```

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 5]
```diff
@@ -399,9 +399,9 @@
     <string name="widgetMode1x1_distance">当前距离</string>     <!-- setting -->
     <string name="widgetMode1x1_apsis">下一个远地点或近地点</string>     <!-- setting -->
 
-    <string name="widgetMode3x1_lightmap_large">Lightmap</string>     <!-- setting -->    <!-- TODO -->
-    <string name="widgetMode3x1_lightmap_medium">Lightmap (medium)</string>     <!-- setting -->    <!-- TODO -->
-    <string name="widgetMode3x1_lightmap_small">Lightmap (small)</string>     <!-- setting -->    <!-- TODO -->
+    <string name="widgetMode3x1_lightmap_large">光照图</string>     <!-- setting -->    
+    <string name="widgetMode3x1_lightmap_medium">光照图 （中等）</string>     <!-- setting -->    
+    <string name="widgetMode3x1_lightmap_small">光照图 （小）</string>     <!-- setting -->
```
Comment (jamesliu96) [inline app/src/main/res/values-zh-rCN/strings.xml:404]: The same.
```suggestion
    <string name="widgetMode3x1_lightmap_small">光照图（小）</string>     <!-- setting -->    
```

**Comment by jamesliu96** [comment]
I'm planning to handle other TODOs in the both Chinese translations after this PR is merged. Hope it could catch the `v0.14.11` release.

**Pull request merged**